### ISSUE TITLE
Where can I find an explanation of the exact kinds of data I can get from the API?

### ISSUE THREAD

**Issue by NathanWailes** [canopy-api/canopy-api#2]
Where can I find an explanation of the exact kinds of data I can get from the API?  I ran the example but it's not clear to me if that was a full list of available fields or if there are other fields I can ask for.

**Comment by Xriuk** [comment]
Yeah, it's not very clear but apparently in the [Playground](https://graphql.canopyapi.co/) there is a docs section with all the objects and fields available:

![Image](https://github.com/user-attachments/assets/4e4422ca-e410-40dd-b3af-ef351663a664)

**Comment by rhino88** [comment]
The image above is correct. Will be adding more docs via open api swagger spec soon.
